### PR TITLE
[WIP] 07 support

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: deps/ext.jl.bak [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# autogenerated file, do not edit
```
Comment (vchuravy) [inline deps/ext.jl.bak]: Don't check this file in.

File: src/array.jl [hunk 2]
```diff
@@ -16,6 +16,8 @@ end
 
 CuArray{T,N}(buf::Mem.Buffer, dims::NTuple{N,Integer}) where {T,N} = CuArray{T,N}(buf, 0, dims)
 
+Base.pointer(A::CuArray) = unsafe_buffer(A)
```
Comment (vchuravy) [inline src/array.jl]: Do we need this?

File: src/utils.jl [hunk 3]
```diff
@@ -26,7 +26,7 @@ function Base.fill!(xs::CuArray, x)
     return
   end
   blk, thr = cudims(xs)
-  @cuda (blk, thr) kernel(xs, convert(eltype(xs), x))
+  @cuda blocks=blk threads=thr kernel(xs, convert(eltype(xs), x))
```
Comment (vchuravy) [inline src/utils.jl]: Probably good to bump the minimal version bound for CUDAnative

File: src/array.jl (hunk 2 above)
Comment (SimonDanisch) [reply src/array.jl]: I thought I relied on that - but I need to assure.

File: deps/ext.jl.bak (hunk 1 above)
Comment (SimonDanisch) [reply deps/ext.jl.bak]: oh, I guess the gitignore needs some tweaking

**Comment by MikeInnes** [comment]
Waiting on https://github.com/JuliaGPU/GPUArrays.jl/pull/108 to push this forward.

**Comment by MikeInnes** [comment]
Closing in favour of #90

**Pull request closed without merging**